docs/mddocs/Quickstart/npu_quickstart.md

```diff
@@ -82,19 +82,42 @@ With the `llm-npu` environment active, use `pip` to install `ipex-llm` for NPU:
 conda activate llm-npu
 
 pip install --pre --upgrade ipex-llm[npu]
+
+:: [optional] for Llama-3.2-1B-Instruct & Llama-3.2-3B-Instruct
+pip install transformers==4.45.0 accelerate==0.33.0
+
+:: [optional] for glm-edge-1.5b-chat & glm-edge-4b-chat
+pip install transformers==4.47.0 accelerate==0.26.0
```

Comment:
Should we put model-specific dependencies in examples instead? :) May be confusing here